libs/application/templates/official-journal-of-iceland/src/components/additions/Additions.tsx

```
@@ -0,0 +1,229 @@
+
+  const onAddAddition = () => {
+    const currentAnswers = structuredClone(currentApplication.answers)
+    const currentAdditions = getValueViaPath(
+      currentAnswers,
+      InputFields.advert.additions,
+    )
+
+    // this should never happen, but how do we handle it?
+    if (!Array.isArray(currentAdditions)) {
+      console.error('Additions must be an array')
+      currentAdditions = []
```

Biome

[error] Can't assign currentAdditions because it's a constant

This is where the variable is defined as constant

Unsafe fix: Replace const with let if you assign it to a new value.


(lint/correctness/noConstAssign)